WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1316,9 +1317,14 @@ public void onImageSettingsRequested(EditorImageMetaData editorImageMetaData) {
     }
 
     @Override public void onPositiveClicked(String instanceTag) {
-        if (instanceTag != null && instanceTag == TAG_PUBLISH_CONFIRMATION_DIALOG) {
-            mPost.setStatus(PostStatus.PUBLISHED.toString());
-            publishPost();
+        if (instanceTag != null) {
+            if (instanceTag == TAG_PUBLISH_CONFIRMATION_DIALOG) {
```

Comment:
This definitely shouldn't block the merge - we may consider adding else branch and throw an exception or log an error message -> to follow the fail ASAP principle. WDYT?

Reply:
Good! Addressed in 731da6f